src/kafkaaggregator/cli.py

```diff
@@ -1,10 +1,105 @@
-"""Entrypoint for Faust CLI."""
+"""Command-line interface for kafkaaggregator."""
 
-__all__ = ["main"]
+__all__ = ["main", "produce", "init_example"]
 
-from kafkaaggregator.app import app
+import asyncio
+import json
+import logging
+import random
+from time import time
+
+from faust.cli import AppCommand, option
+
+from kafkaaggregator.app import app, config
+from kafkaaggregator.generator import AgentGenerator
+from kafkaaggregator.topics import SourceTopic
+
+logger = logging.getLogger("kafkaaggregator")
 
 
 def main() -> None:
     """Entrypoint for Faust CLI."""
     app.main()
+
+
+@app.command(
+    option(
+        "--frequency",
+        type=float,
+        default=10,
+        help="The frequency in Hz in wich messages are produced.",
+        show_default=True,
+    ),
+    option(
+        "--max-messages",
+        type=int,
+        default=600,
+        help="The maximum number of messages to produce.",
+        show_default=True,
+    ),
+)
+async def produce(
+    self: AppCommand, frequency: float, max_messages: int
+) -> None:
+    """Produce messages for the kafkaaggregator source topic.
+
+    Parameters
```

Comment:
Because Click uses the docstring as the help text for the CLI, the numpydoc standard doesn't appy here. I'd delete all the Parameters documentation and focus on writing help text as you'd expect it for CLI.